### PR TITLE
Update urls

### PR DESCRIPTION
**Pull request by stuartmoore1023**
per #360 PTAL

### CODE REVIEW

File: jekyll/_includes/airbrake-js-going-further.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
 Installation and configuration is just the beginning. The [airbrake-js
 notifier](https://github.com/airbrake/airbrake-js) supports many other advanced uses and options including:
 
-- [adding extra details to errors](https://github.com/airbrake/airbrake-js#notice-annotations)
-- [sourcemaps for easy to parse backtraces](https://github.com/airbrake/airbrake-js#source-map)
-- [filtering errors](https://github.com/airbrake/airbrake-js#filtering-errors)
-- [specifying error severity](https://github.com/airbrake/airbrake-js#severity)
-- [adding custom reporters](https://github.com/airbrake/airbrake-js#custom-reporters)
+- [adding extra details to errors](https://github.com/airbrake/airbrake-js/tree/master/packages/browser#notice-annotations)
+- [sourcemaps for easy to parse backtraces](https://github.com/airbrake/airbrake-js/tree/master/packages/browser#source-map)
```
Comment (thompiler) [inline jekyll/_includes/airbrake-js-going-further.md]: I think the section is "Source maps" now so it should have an "s" at the end of the URL.

File: jekyll/_includes/airbrake-js-going-further.md [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
+- [filtering errors](https://github.com/airbrake/airbrake-js/tree/master/packages/browser#filtering-errors)
+- [specifying error severity](https://github.com/airbrake/airbrake-js/tree/master/packages/browser#severity)
+- [adding custom reporters](https://github.com/airbrake/airbrake-js/tree/master/packages/browser#custom-reporters)
```
Comment (thompiler) [inline jekyll/_includes/airbrake-js-going-further.md]: Looks like custom reporters were removed:
https://github.com/airbrake/airbrake-js/commit/67ae78a01f7a4bcdbdbc8ac5cbdf6bae338347cc

Let's just remove this line too then.

File: jekyll/_includes/airbrake-js-going-further.md (hunk 1 above)
Comment (stuartmoore1023) [reply jekyll/_includes/airbrake-js-going-further.md]: Done

File: jekyll/_includes/airbrake-js-going-further.md (hunk 2 above)
Comment (stuartmoore1023) [reply jekyll/_includes/airbrake-js-going-further.md]: Done.

**Pull request merged**